specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/common/2022-07-01/definitions.json

```diff
@@ -0,0 +1,168 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-07-01",
```

Comment:
Why do you need a separate folder for common if you plan to rev it in each new api version ?


Eitherways , please also copy the previous version files for common in the first commit like you did for the main folder.

Reply:
For now can you just commit the previous version in one commit and then add this file as the next commit over it and let me know so that I can do a diff ?

If you have made absolutely no changes between this and the previous version from  2022-03-01 then just let me know in the comments and I can sign off (This comment is applicable for definitions.json and parameters.json)

Reply:
Addressing both comments here.
When going through the swagger process, OpenAPI duplicates files for you into a new folder and also changed references from common/2022-03-01 to common/2022-07-01 (the first 3 commits). Thus, I duplicated the common folder so that these references could be found. 

Since I am not changing the definitions.json and parameters.json, I agree that it makes more sense to use the previous common version. I've reflected this in my latest commit.